Lib/traceback.py

```diff
@@ -621,3 +624,83 @@ def format(self, *, chain=True):
             yield 'Traceback (most recent call last):\n'
         yield from self.stack.format()
         yield from self.format_exception_only()
+
+
+class TracebackExceptionGroup:
+    """An exception or exception group ready for rendering.
+
+    The traceback module captures enough attributes from the original exception
+    group to this intermediary form to ensure that no references are held, while
+    still being able to fully print or format it.
+
+    Use `from_exception` to create TracebackExceptionGroup instances from exception
+    objects, or the constructor to create TracebackExceptionGroup instances from
+    individual components.
+
+    - :attr:`summary` A tuple of (indentation level, TracebackException) pairs
```

Comment:
Can you explain more about the structure of this attriubute? A brief example might suffice.

Reply:
I wanted to discuss it - how do you feel about adding the indent field to TracebackException (defaulted to 0) and then this is just a list of TracebackExceptions?

Reply:
Isn’t that a namedtuple? We can’t add fields to those since people might unpack them into fixed targets.

Reply:
No, it's just a class.

Reply:
Oh, then that seems to be a nice improvement.